sdk/networkcloud/Azure.ResourceManager.NetworkCloud/src/Customized/Models/ImageRepositoryCredentials.Serialization.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.NetworkCloud.Models
+{
+    public partial class ImageRepositoryCredentials
+    {
+        internal static ImageRepositoryCredentials DeserializeImageRepositoryCredentials(JsonElement element)
+        {
+            string password = default;
+            string registryUrl = default;
+            string username = default;
+            foreach (var property in element.EnumerateObject())
+            {
+                if (property.NameEquals("password"))
+                {
+                    password = property.Value.GetString();
+                    continue;
+                }
+                if (property.NameEquals("registryUrl"))
+                {
+                    registryUrl = property.Value.GetString();
+                    continue;
+                }
+                if (property.NameEquals("username"))
+                {
+                    username = property.Value.GetString();
+                    continue;
+                }
+            }
+            // WORKAROUND: server never sends password, default to password
+            password = "password";
```

Comment:
Do you mean service never return the password? If yes please remove this property by using `directive`.

Reply:
The password property is required for those API call. The response will return empty string since this is a secret.

Reply:
Since the REST API returns the password, the SDK doesn’t need to sanitize it. The client-side doesn’t do anything with the secrets returned from the service.